examples/python/pincell_depletion/restart_depletion.py

```diff
@@ -0,0 +1,84 @@
+
+chain_file = './chain_simple.xml'
+power = 174 # W/cm, for 2D simulations only (use W for 3D)
+
+###############################################################################
+#                      Load previous simulation results
+###############################################################################
+
+# Load geometry from statepoint
+statepoint = 'statepoint.100.h5'
+sp = openmc.StatePoint(statepoint)
+geometry = sp.summary.geometry
+
+# Close statepoint and summary files to be able to write over them
+sp.close()
+
+# Load previous depletion results
+previous_results = openmc.deplete.ResultsList("depletion_results.h5")
+
+###############################################################################
+#                      Transport calculation settings
+###############################################################################
+
+# Instantiate a Settings object, set all runtime parameters
+settings_file = openmc.Settings()
+settings_file.batches = batches
+settings_file.inactive = inactive
+settings_file.particles = particles
+
+# Create an initial uniform spatial source distribution over fissionable zones
+bounds = [-0.62992, -0.62992, -1, 0.62992, 0.62992, 1]
+uniform_dist = openmc.stats.Box(bounds[:3], bounds[3:], only_fissionable=True)
+settings_file.source = openmc.source.Source(space=uniform_dist)
+
+entropy_mesh = openmc.Mesh()
+entropy_mesh.lower_left = [-0.39218, -0.39218, -1.e50]
+entropy_mesh.upper_right = [0.39218, 0.39218, 1.e50]
+entropy_mesh.dimension = [10, 10, 1]
+settings_file.entropy_mesh = entropy_mesh
+
+###############################################################################
+#                   Initialize and run depletion calculation
+###############################################################################
+
+op = openmc.deplete.Operator(geometry, settings_file, chain_file, \
+                                                  previous_results)
+
+# Perform simulation using the predictor algorithm
+openmc.deplete.integrator.predictor(op, time_steps, power)
+
+###############################################################################
+#                    Read depletion calculation results
+###############################################################################
+
+# Open results file
+results = openmc.deplete.ResultsList("depletion_results.h5")
+
+# Obtain K_eff as a function of time
+time, keff = results.get_eigenvalue()
+                                                                                                                                                                                                                                                                                                                                                                                                                                                        
+# Plot eigenvalue as a function of time
+plt.figure()
+plt.plot(time/24/60/60, keff, label="K-effective")
```

Comment:
`time/(24*60*60)`

Fun fact: the Python compiler performs [constant folding](https://en.wikipedia.org/wiki/Constant_folding) so that when your code is actually run, the interpreter only sees the single constant 86400. Here's a disassembler proof:
```Python
>>> import dis
>>> dis.dis('time/(24*60*60)')
  1           0 LOAD_NAME                0 (time)
              2 LOAD_CONST               3 (86400)
              4 BINARY_TRUE_DIVIDE
              6 RETURN_VALUE
```